assisted-events-scrape/storage/__init__.py

```diff
@@ -0,0 +1 @@
+from .cluster_events_storage import ClusterEventsStorage
```

Comment:
Need to declare `__all__` for defining what symbols in a module will be exported